pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -3118,6 +3118,22 @@ let
 
       ms-vscode-remote.remote-ssh = callPackage ./ms-vscode-remote.remote-ssh { };
 
+      ms-vscode-remote.remote-ssh-edit = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "remote-ssh-edit";
+          publisher = "ms-vscode-remote";
+          version = "0.47.2";
```

```suggestion
          version = "0.86.0";
```
Marketplace version is at `0.86.0`. Is version right?

---

Not sure where I got 0.47.2 from. Will update.